aiinfra-cluster/modules/slurm-cluster/main.tf

```diff
@@ -0,0 +1,71 @@
+/**
+  * Copyright 2022 Google LLC
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+module "compute_node_group" {
+  source = "github.com/GoogleCloudPlatform/hpc-toolkit//community/modules/compute/schedmd-slurm-gcp-v5-node-group//?ref=develop"
+
+  project_id             = var.project_id
+  labels                 = var.labels
+  node_count_static      = var.node_count_static
+  node_count_dynamic_max = var.node_count_dynamic_max
+  instance_template      = var.instance_template_compute
+}
+
+module "compute_partition" {
+  source = "github.com/GoogleCloudPlatform/hpc-toolkit//community/modules/compute/schedmd-slurm-gcp-v5-partition//?ref=develop"
+
+  deployment_name      = var.deployment_name
+  project_id           = var.project_id
+  region               = var.region
+  partition_name       = "compute"
+  subnetwork_self_link = var.subnetwork_self_link
+  network_storage      = flatten([var.network_storage])
+  node_groups          = flatten([module.compute_node_group.node_groups])
+  enable_placement     = false
+}
+
+module "slurm_controller" {
+  source = "github.com/GoogleCloudPlatform/hpc-toolkit//community/modules/scheduler/schedmd-slurm-gcp-v5-controller//?ref=develop"
```

Comment:
we should use a specific SHA as ref. keeping ref=develop would cause unintentional code change in HPCToolkit repo to break us.

Reply:
I put it as ref=v1.16.0 (which is also where HEAD is rn) so that it is a little more readable. is that cool or is SHA a better idea?

Reply:
should be good.